src/Database/DriverInterface.php

```diff
@@ -97,23 +97,23 @@ public function rollbackTransaction(): bool;
     /**
      * Get the SQL for releasing a save point.
      *
-     * @param string|int $name The table name.
+     * @param string|int $name Save point name or id
      * @return string
      */
     public function releaseSavePointSQL($name): string;
```

Comment:
I don't know why but these take string|int instead of just string. It didn't seem right changing this to $savePoint because we pass in the name instead of generating the save point to use elsewhere.